src/reagent/impl/template.cljs

```diff
@@ -73,13 +73,23 @@
                            class))))
     p))
 
+(defn stringify-class [{:keys [class] :as props}]
+  (if (coll? class)
```

Comment:
Supporting sets is good, so just leave it that way.
I guess the remark meant to ask how to deal with maps. Maybe throwing an
error would be best, but probably "undefined behavior" is okay too - I
mean, who would do that anyways :-P

Adrian Gruntkowski <notifications@github.com> schrieb am Mi., 28. Okt. 2015
10:19:

> In src/reagent/impl/template.cljs
> <https://github.com/reagent-project/reagent/pull/154#discussion_r43231780>
> :
>
> > @@ -73,13 +73,23 @@
> >                             class))))
> >      p))
> >
> > +(defn stringify-class [{:keys [class] :as props}]
> > +  (if (coll? class)
>
> I have used coll? in order to support sets actually (which is not covered
> by tests as I see it now, bummer). Do you think it's
> impractical/unnecessary?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/reagent-project/reagent/pull/154/files#r43231780>.
>